CHANGELOG.md

```diff
@@ -1,6 +1,33 @@
 Change Log / Release Log for mallocMC
 ================================================================
 
+2.5.0crp
+--------
+**Date:** 2021-02-XX
```

Comment:
Do we have a date for the release already?

Reply:
not yet, I think next week would be week to release^^

Reply:
Let us wait with merging until I have a release date